packages/sit-onyx/src/styles/variables-onyx.css

```diff
@@ -0,0 +1,164 @@
+/**
+ * Do not edit directly.
+ * This file contains the specific variables for the "onyx" theme.
+ * Imported from Figma API on Tue, 09 Jan 2024 08:17:34 GMT
+ */
+:root {
+  --onyx-color-brand-primary-100: #e9f7f1;
+  --onyx-color-brand-primary-200: #caf0df;
+  --onyx-color-brand-primary-300: #a0e7c8;
+  --onyx-color-brand-primary-400: #76deb0;
+  --onyx-color-brand-primary-500: #4cd598;
+  --onyx-color-brand-primary-600: #42bc85;
+  --onyx-color-brand-primary-700: #37a373;
+  --onyx-color-brand-primary-800: #2d8a60;
+  --onyx-color-brand-primary-900: #22714d;
+  --onyx-color-brand-primary-1000: #18583a;
+  --onyx-color-brand-primary-1100: #0d3f28;
+  --onyx-color-brand-primary-1200: #032615;
+  --onyx-color-brand-secondary-100: var(--onyx-color-brand-primary-100);
+  --onyx-color-brand-secondary-200: var(--onyx-color-brand-primary-200);
+  --onyx-color-brand-secondary-300: var(--onyx-color-brand-primary-300);
+  --onyx-color-brand-secondary-400: var(--onyx-color-brand-primary-400);
+  --onyx-color-brand-secondary-500: var(--onyx-color-brand-primary-500);
+  --onyx-color-brand-secondary-600: var(--onyx-color-brand-primary-600);
+  --onyx-color-brand-secondary-700: var(--onyx-color-brand-primary-700);
+  --onyx-color-brand-secondary-800: var(--onyx-color-brand-primary-800);
+  --onyx-color-brand-secondary-900: var(--onyx-color-brand-primary-900);
+  --onyx-color-brand-secondary-1000: var(--onyx-color-brand-primary-1000);
+  --onyx-color-brand-secondary-1100: var(--onyx-color-brand-primary-1100);
+  --onyx-color-brand-secondary-1200: var(--onyx-color-brand-primary-1200);
+  --onyx-color-brand-neutral-100: #fafbfc;
+  --onyx-color-brand-neutral-200: #cad0d5;
+  --onyx-color-brand-neutral-300: #9ba5ad;
+  --onyx-color-brand-neutral-400: #6b7a86;
+  --onyx-color-brand-neutral-500: #465a69;
+  --onyx-color-brand-neutral-600: #334654;
+  --onyx-color-brand-neutral-700: #2a3c4a;
+  --onyx-color-brand-neutral-800: #223340;
+  --onyx-color-brand-neutral-900: #192a37;
+  --onyx-color-brand-neutral-1000: #11212d;
+  --onyx-color-brand-neutral-1100: #081723;
+  --onyx-color-brand-neutral-1200: #000e19;
+  --onyx-color-universal-grey-white: #ffffff;
```

Comment:
Suggestion: Keep namings flat and remove the `universal` group, so:
`--onyx-color-grey-100`
`--onyx-color-red-100`
`--onyx-color-green-100`
etc.

Reply:
We need the separation between brand-colors and universal colors. I can give more details about the token structure in the next meeting.